chalicelib_fourfront/checks/wrangler_checks.py

```diff
@@ -1105,7 +1105,7 @@ def users_with_pending_lab(connection, **kwargs):
     time.sleep(wait)
     check.action = 'finalize_user_pending_labs'
     check.full_output = []
-    check.status = 'PASS'
+    check.status = 'WARN'
```

Comment:
I don't see anywhere that this check has a 'PASS' status.  Is that intentional?  Maybe in the ELSE along with `check.summary = 'No users found with pending_lab'`